src/Welcome.md

```diff
@@ -0,0 +1,23 @@
+# Getting Started
```

Comment:
Love the flow and basic presentation. I added comments elsewhere in the PR that apply to this welcome.md as well.  Most notably: we need to be inclusive of more than just the ODP EC firmware and for the EC part, should reference a hardware platform we support today.  We can discuss...